web/template/login.gohtml

```diff
@@ -41,7 +41,7 @@
                 <div class="text-sm">
                     <label for="password" class="block text-5">Password</label>
                     <input type="password" name="password" id="password" required
-                           class="tl-input mt-3"
+                           class="tl-input mt-3" autocomplete=""
```

Comment:
Why not use 
```suggestion
                           class="tl-input mt-3" autocomplete="current-password"
```
As suggested by the browser?

Reply:
Thanks. Apparently I can't read. 🫠